### ISSUE TITLE
target_size width/height order is swapped between evaluation script and gridy2gridx_flow — deforms non-square crops

### ISSUE THREAD

**Issue by DaniVerAusil** [Egkang-Luis/BurstM#4]
Thanks for releasing the code and checkpoints, very useful work!

I believe there's a real width/height ordering bug that only shows up for non-square inputs, which is why it's not caught by the provided evaluation scripts, since they only benchmark square 80×80 crops.

Where the mismatch is:

models/utils.py: 38-51 --> gridy2gridx_flow(gridy, th, tw, offset, ...) uses th/tw positionally as F.interpolate(size=(th, tw)), i.e. it expects target_size = [height, width].
models/BurstM.py: 126-132 --> Neural_Warping.forward calls utils.gridy2gridx_flow(coord, *target_size, ...), unpacking target_size positionally into (th, tw).
BurstM_Track_2_evaluation.py: 89-90 (and again at : 153-154) -->  builds target_size as:

target_size[0] = int(burst.shape[-1] * float(args.scale))  # this is WIDTH (last dim)
target_size[1] = int(burst.shape[-2] * float(args.scale))  # this is HEIGHT

i.e. target_size = [width*scale, height*scale].

So gridy2gridx_flow receives th = width*scale and tw = height*scale (swapped). 
For a square crop th == tw numerically, so it's invisible; for any rectangular crop, the output comes out with the aspect ratio inverted (a wide crop becomes tall, and vice versa).

Minimal repro (using a rectangular packed burst, h=24, w=48, so the expected output H/W ratio is 0.5):

target_size = [int(w * scale), int(h * scale)]          # current convention
out = model(burst_ref, burst_src, scale, target_size)
print(out.shape)   # (1, 3, 384, 192) -> H/W = 2.0  (WRONG, inverted)

target_size = [int(h * scale), int(w * scale)]           # swapped
out = model(burst_ref, burst_src, scale, target_size)
print(out.shape)   # (1, 3, 192, 384) -> H/W = 0.5  (correct, matches input ratio)

**Suggested fix**: either swap the assignment order in the evaluation scripts (target_size[0] should be height*scale, target_size[1] should be width*scale) to match what gridy2gridx_flow actually consumes, or (probably clearer long-term) rename the parameters in gridy2gridx_flow/Neural_Warping.forward and document explicitly which order target_size is expected in, since right now the two conventions silently disagree.

Thank u again for your work I found it very interesting and usefull!!
